### PR TITLE
Update cloud user limits

### PR DESCRIPTION
**Pull request by justinegeffen**
Updates for https://github.com/mattermost/mattermost-webapp/pull/10061.

### CODE REVIEW

**Review by jasonblais — APPROVED** [review]

File: source/about/cloud-subscriptions.rst [hunk 1]
```diff
@@ -46,9 +46,9 @@ Frequently asked questions
 What happens when my 14-day trial period ends?
```
Comment (cwarnermm) [inline source/about/cloud-subscriptions.rst:46]: When Cloud Freemium is released, is there going to be a 14-day trial period?

File: source/about/cloud-subscriptions.rst [hunk 2]
```diff
@@ -85,14 +85,14 @@ How is Mattermost Cloud secured?
 
 Mattermost Cloud has encryption at-rest and in-transit. We're SOC2 Type 1 certified and working towards SOC2 Type 2.
 
-Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC (Virtual Private Cloud) dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace. Please contact Mattermost Sales for more information on this option. 
+Mattermost Cloud Enterprise is available to be deployed in a private environment within an AWS VPC (Virtual Private Cloud) dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace. Please contact Mattermost Sales for more information on this option.
```
Comment (cwarnermm) [inline source/about/cloud-subscriptions.rst:88]: is it "on-premises" or "on-premise"?

File: source/manage/cloud-billing.rst [hunk 3]
```diff
@@ -17,7 +17,7 @@ Mattermost Cloud is offered as a pay-as-you-go subscription service.
 
 You will be billed for registered users in your workspace. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they are not deactivated in the billing period.
 
-Customers who signed up for a Mattermost Cloud workspace from our October 2020 launch can continue to use their Cloud instance free of charge for up to ten registered users; however, these customers must upgrade to a per-user pricing model after reaching 11 or more registered users. New Mattermost Cloud workspaces can trial Mattermost Cloud free for 14 days before being required to upgrade to a monthly per-user subscription. 
+Mattermost Cloud workspaces can trial Mattermost Cloud free for 14 days before being required to upgrade to a monthly per-user subscription.
```
Comment (cwarnermm) [inline source/manage/cloud-billing.rst:20]: 14-day trial question applies here too

**Review by cwarnermm — COMMENTED** [review]
Feedback (in the form of questions to consider) provided inline.

File: source/about/cloud-subscriptions.rst (hunk 2 above)
Comment (justinegeffen) [reply source/about/cloud-subscriptions.rst:88]: On-premises is the correct way of saying it, but we say "on-premise" in other parts of the docs, and we also say "on-prem" (which is a nice compromise). I'd propose going with on-prem as that's on the website and we can reverse engineer any other instances. Thoughts?

File: source/about/cloud-subscriptions.rst (hunk 1 above)
Comment (justinegeffen) [reply source/about/cloud-subscriptions.rst:46]: Good question. I'll need to check but for now let's leave this as-is and I have noted this down as needing an update/review for 7.0 docs.

**Pull request merged**